modules/virt-define-guest-agent-ping-probe.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+:_content-type: PROCEDURE
+[id="virt-define-guest-agent-ping-probe_{context}"]
+
+= Defining a guest agent ping probe
+
+Define a guest agent ping probe by setting the `spec.readinessProbe.guestAgentPing` field of the virtual machine instance (VMI) configuration.
+
+:FeatureName: The guest agent ping probe
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* The QEMU guest agent must be installed and enabled on the virtual machine.
+
+.Procedure
+
+. Include details of the guest agent ping probe in the VMI configuration file.
++
+
+.Sample guest agent ping probe
+[source,yaml]
+----
+# ...
+spec:
+  readinessProbe:
+    guestAgentPing: {} <1>
+    initialDelaySeconds: 120 <2>
+    periodSeconds: 20 <3>
+    timeoutSeconds: 10 <4>
+    failureThreshold: 3 <5>
+    successThreshold: 3 <6>
+# ...
+----
+<1> The guest agent ping probe to connect to the VMI.
```

Comment:
Maybe include what is optional and what is mandatory

Reply:
Good idea, so we just have to gather the information to write it similar to https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

Reply:
Looks good, just for reference: https://github.com/kubevirt/kubevirt/blob/3db34971b2df184de40f88c56cd392f356621cc9/staging/src/kubevirt.io/api/core/v1/defaults.go#L158